CHANGELOG.rst

```diff
@@ -17,6 +17,7 @@ New features and enhancements
 * Added a new ``xclim.indices.generic.bivariate_count_occurrences`` function to count instances where operations and performed and validated for two variables. (:pull:`2030`).
 * `xclim` now tracks energy usage and carbon emissions ("last run", "average", and "total") during CI workflows using the `eco-ci-energy-estimation` GitHub Action. (:pull:`2046`).
 * ``xclim.testing.helpers.test_timeseries`` now accepts a `calendar` argument that is forwarded to ``xr.cftime_range``. (:pull:`2019`).
+* ``xclim.indicators.atmos`` now exports the FAO-56 Penman-Monteith equation as `fao_allen98` (:issue:`2004`, :pull:`2067`).
```

Comment:
Did you mean `xclim.indices` or did you want to also make an indicator out of the new function ?

Reply:
Fixed